src/formats/toml.js

```diff
@@ -4,13 +4,19 @@ import moment from 'moment';
 import AssetProxy from 'ValueObjects/AssetProxy';
 import { sortKeys } from './helpers';
 
+// IE polyfill for Number.isInteger
```

Comment:
We don't actually support IE, so this can be removed. 😃

Reply:
Docs refer to IE11 [here](https://www.netlifycms.org/docs/add-to-your-site/#add-the-netlify-identity-widget). Couldn't find any other references to browser support in docs though.

...Will remove this polyfill shortly though, as I can confirm that the CMS won't load at all in IE11 right now.

Reply:
...BTW, I did find the discussion in issues [here](https://github.com/netlify/netlify-cms/issues/674#issuecomment-338775684) RE: browser support, with a summary at the end. 😃

Reply:
Haha, feel free to PR a removal for that if you want.

cc/ @verythorough